.infrastructure/pvc.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-todoapp
+  namespace: todoapp
+spec:
+  storageClassName: standard
+  volumeName: pv-todoapp
+  accessModes:
+    - ReadWriteMany
+  resouces:
```

Comment:
The field `resouces` is misspelled. It should be corrected to `resources` to properly specify the resource requests for the PersistentVolumeClaim.